scripts/config-doc-command.js

```diff
@@ -1,21 +1,73 @@
 const { definitions } = require('../lib/utils/config/index.js')
+const usageFn = require('../lib/utils/usage.js')
 const { writeFileSync, readFileSync } = require('fs')
 const { resolve } = require('path')
 
 const configDoc = process.argv[2]
 const commandFile = process.argv[3]
 
+const TAGS = {
+  CONFIG: {
+    START: '<!-- AUTOGENERATED CONFIG DESCRIPTIONS START -->',
+    END: '<!-- AUTOGENERATED CONFIG DESCRIPTIONS END -->',
+  },
+  USAGE: {
+    START: '<!-- AUTOGENERATED USAGE DESCRIPTIONS START -->',
+    END: '<!-- AUTOGENERATED USAGE DESCRIPTIONS END -->',
+  },
+}
+
 // Note: commands without params skip this whole process.
-const { params } = require(resolve(commandFile))
+const {
+  params,
+  usage,
+} = require(resolve(commandFile))
 
-const describeAll = () =>
-  params.map(name => definitions[name].describe()).join(
+const describeAll = (content) =>
+  content.map(name => definitions[name].describe()).join(
     '\n\n<!-- automatically generated, do not edit manually -->\n' +
       '<!-- see lib/utils/config/definitions.js -->\n\n'
   )
 
-const addBetweenTags = (doc, startTag, endTag, body) => {
+const describeUsage = ({ usage }) => {
+  const synopsis = []
+  const commandName = commandFile.split('/').pop().split('.')[0]
+  synopsis.push('\n```bash')
+
+  if (commandName) {
+    let baseCommand = `npm ${commandName}`
+
+    // special case for `npx`
+    if (commandName === 'exec') {
```

Comment:
```suggestion
    if (commandName === 'npx') {
```

Reply:
I *think* this does what we want. The `npm-exec.md` file should show `npm exec` so it doesn't need a special case.  the `npx.md` file is special because it's not technically a command in and of itself, so it just needs the usage and parameters of `npm exec`, and none of the aliases

Reply:
Oh! I see. I am sorry I confused both commands 😅